### PR TITLE
News cards feed

### LINKED ISSUE #9: News feed in cards block
Latest news articles must appear in cards bock on the home page ordered by date in descending order.
See https://www.sunstar.com/
image

### PR DESCRIPTION
**Pull request by sdmcraft**
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fixes #9 

Test URLs:
- Original: https://www.sunstar.com/<path>
- Before: https://i-9c--sunstar--hlxsites.hlx.live/_drafts/satyam/
- After: https://news-cards--sunstar--hlxsites.hlx.live/_drafts/satyam/

- [ ]  **If you are adding a new block or a variation to an existing block**, please fill below:

  Block library path: https://<branch>--sunstar--hlxsites.hlx.page/tools/sidekick/library.html?plugin=blocks&path=/sidekick/blocks/<your-block>&index=0

### CODE REVIEW

File: scripts/scripts.js [hunk 1]
```diff
@@ -560,3 +561,15 @@ export async function loadConsentManager() {
...
   window.dispatchEvent(new CustomEvent('consentmanager'));
 }
+
+/**
+ * Formats a unix timestamp into a date string.
+ * Example: 1609459200 -> January 1, 2021
+ * @param {*} unixTimestamp
+ * @returns {string} The formatted date string
+ */
+export function formatDateFromUnixTimestamp(unixTimestamp) {
```
Comment (jindaliiita) [inline scripts/scripts.js:571]: i think we should maintain consistency with date format across the whole website.
For that u can change logic of [getFormattedDate function](https://github.com/hlxsites/sunstar/blob/main/scripts/lib-franklin.js#L693) and also create a wrapper for this function which takes timestamp as input.

**Review by dnbute — COMMENTED** [review]
Is the fact that the cards are not clickable expected for this pr?

**Comment by sdmcraft** [comment]
> Is the fact that the cards are not clickable expected for this pr?

Fixed.

**Comment by sdmcraft** [comment]
> Is the fact that the cards are not clickable expected for this pr?

Fixed.

**Comment by sdmcraft** [comment]
> ### 🔸 2 visual differences detected
> * **/sidekick/blocks/cards** ([main](https://main--sunstar--hlxsites.hlx.page/sidekick/blocks/cards) vs [branch](https://news-cards--sunstar--hlxsites.hlx.page/sidekick/blocks/cards))Expected an image 1280px by 1175px, received 1280px by 1011px. 39818 pixels (ratio 0.03 of all image pixels) are different.
> * **/sidekick/blocks/sections** ([main](https://main--sunstar--hlxsites.hlx.page/sidekick/blocks/sections) vs [branch](https://news-cards--sunstar--hlxsites.hlx.page/sidekick/blocks/sections))Expected an image 1312px by 6530px, received 1312px by 6512px.
> 
> The diff images are [attached in the artifact](https://github.com/hlxsites/sunstar/actions/runs/6404224382)

Both of these are expected since this PR proposes changes in `cards` styling.

File: blocks/cards/cards.css [hunk 2]
```diff
@@ -93,11 +99,9 @@ main .block.cards {
 }
 
 .cards .cards-card-image {
+  height: 155px;
```
Comment (hparwani2) [inline blocks/cards/cards.css:102]: can we use rem here?

**Pull request merged**